.github/workflows/build-unified.yml

```diff
@@ -158,6 +162,36 @@ jobs:
           secrets: |
             "GH_PAT=${{ steps.retrieve-secret-pat.outputs.github-pat-bitwarden-devops-bot-repo-scope }}"
 
+      - name: Install Cosign
+        if: inputs.is_workflow_call == true && github.ref == 'refs/heads/main'
```

Comment:
⚠️ Please check me here. I want this and the step below to only run when this container is built for main branch pushes in `server` -- not sure how feasible that is.

Reply:
We haven't been passing this input `is_workflow_call` from the trigger job in server, I will make an update to server. Other than that the `if` check is alright

Reply:
Please hold on merging this so I can merge the change on Server first

Reply:
Can be merged now